src/graphql/getters/getSeasonBadges/helpers.ts

```diff
@@ -1,82 +1,26 @@
-import { BadgeVariant } from './getSeasonBadges';
 import { Season } from '../../../hooks/season/types';
+import { BadgeVariant, GetProfileBadgesResponse } from './types';
+import { SEASON_BADGE_VARIANTS } from './mappers';
 
-export const BADGE_TYPE_BADGE_VARIANT_MAP: Record<string, BadgeVariant> = {
-  // season 1
-  '12': 'fixedTrader',
-  '13': 'deltaDegen',
-  '14': 'leverageCrowbar',
-  '15': 'irsConnoisseur',
-  '16': 'sushiRoll',
-  '17': 'degenStuff',
-  '18': 'okBoomer',
-  '19': 'lpoor',
-  '20': 'moneyMoneyMoney',
-  '21': 'waterHose',
-  '22': 'rainMaker',
-  '23': 'dryIce',
-  '24': 'beWaterMyFriend',
-  '25': 'yikes',
-  '26': 'maxBidding',
-  '27': 'topTrader',
-  // season OG
-  '0': 'ogFixedTrader',
-  '1': 'ogDeltaDegen',
-  '2': 'ogLeverageCrowbar',
-  '3': 'ogIrsConnoisseur',
-  '4': 'ogSushiRoll',
-  '5': 'ogDegenStuff',
-  '6': 'ogOkBoomer',
-  '7': 'ogLpoor',
-  '8': 'ogMoneyMoneyMoney',
-  '9': 'ogWaterHose',
-  '10': 'ogRainMaker',
-  '11': 'ogDryIce',
-  '28': 'ogBeWaterMyFriend',
-  '29': 'ogYikes',
-  '30': 'ogMaxBidding',
-  '31': 'ogTopTrader',
-};
+export function getDefaultResponse(seasonId: Season['id']): GetProfileBadgesResponse {
+  return SEASON_BADGE_VARIANTS[seasonId].map((b) => ({
+    variant: b as BadgeVariant,
+  }));
+}
+
+/**
+ * "Convert seconds to milliseconds, but only if the input is a number and not zero."
+ *
+ * @param {number} seconds - number - The number of seconds to convert to milliseconds.
```

Comment:
nice comments! very helpful

Reply:
thanks I plan to add them to other utilities as I go through codebase